<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="ConnectBtn">
     <property name="text">
      <string>Conect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ZeroBtn">
     <property name="text">
      <string>Zero</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="StartRecordBtn">
     <property name="text">
      <string>Start Recording</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="StopRecordBtn">
     <property name="text">
      <string>Stop Recording</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
